docs/content/topics/workers.md

```diff
@@ -9,64 +9,225 @@ aliases:
+  document](dependencies.md) for a detailed description of this approach.
 
-The Brigade worker (`brigade-worker`) is captured in a Docker image, and that
-image is then executed in a container on your cluster. Augmenting the worker,
-then, is done by creating a custom Docker image and then configuring Brigade
-to use that image instead of the default `brigade-worker` image.
+1. Create a custom Docker image for the worker that contains the additional
+   dependencies.
 
-Next we will see how to quickly create a custom worker by creating a new
-Docker image based on the base image.
+The remainder of this section focuses on the second approach.
 
-## Creating a Custom Worker
+Since the Brigade worker (`brigade-worker`) is supplied as a Docker image,
+amending the default worker with additional Node.js libraries is as simple as
+"extending" the default worker's Docker image to create a custom image.
 
-As we saw above, workers are Docker images. And the default worker is a Docker
-image that contains the Brigade worker runtime, which can read and execute
-`brigade.js` files.
+By way of example, suppose we wish to provide our `brigade.js` with access to
+an XML parser library. This can be accomplished using the following
+`Dockerfile`:
 
-At its core, the Brigade worker is just a Node.js application. That means that
-we can use the usual array of Node.js tools and libraries. Here, we'll load an
-extra library from NPM so that it is available inside of our `brigade.js`
-scripts.
+```Dockerfile
+FROM brigadecore/brigade-worker:v1.1.0
 
-Since the main worker is already tooled to do the main processing, the easiest
-way to add your own libraries is to start with the existing worker and add to
-it. Docker makes this convenient.
+RUN yarn add xml-simple
+```
 
-Say we want to provide an XML parser to our Brigade scripts. We can do that
-using a `Dockerfile`:
+The `Dockerfile` begins with the default Brigade worker image and simply adds
+the `xml-simple` library.
 
-```Dockerfile
-FROM brigadecore/brigade-worker:latest
+Next, skip to the section on [building and publishing a custom worker image](#building-and-publishing-a-custom-worker-image).
+
+## Adding a Custom (Non-NPM) Library
+
+Sometimes it is useful to encapsulate commonly used Brigade code into a library
+that can be shared between projects internally. While the NPM model above is
+easier to manage over the longer term, there is a simple method for loading
+custom code into an image. This section illustrates that method.
+
+Here is a small library that adds an `alpineJob()` helper function:
+
+[mylib.js](examples/workers/mylib.js)
+```javascript
+const {Job} = require("./brigadier");
+
+exports.alpineJob = function(name) {
+  j = new Job(name, "alpine:3.7", ["echo hello"])
+  return j
+}
+```
+
+**Note:** Because we are loading our code directly into Brigade, we import
+`./brigadier`, not `brigadier`. This may change in the future.
```

Comment:
Not sure I'm following this - the default implementation of the worker takes care of this, and allows importing `@brigadecore/brigadier` as well.

Reply:
That line was copied verbatim from the previous version of this doc. This PR just moves it. I'm all for removing old/superfluous details. If you think this can go, I'm happy to remove it.